enterprise/server/remote_execution/containers/podman/podman.go

```diff
@@ -927,6 +927,9 @@ func (c *podmanCommandContainer) State(ctx context.Context) (*rnpb.ContainerStat
 func runPodman(ctx context.Context, subCommand string, stdio *container.Stdio, args ...string) *interfaces.CommandResult {
 	command := []string{
 		"podman",
+		// Use transient store to reduce contention.
+		// See https://github.com/containers/podman/issues/19824
+		"--transient-store",
```

Comment:
do you think it's a good idea to put this behind a flag so we can test in dev for a little longer? (rather than just one day)

Reply:
yeah, good call - Done